packages/flutter/lib/src/material/slider.dart

```diff
@@ -643,25 +639,25 @@ class _RenderSlider extends RenderBox {
       _state.overlayController.forward();
       if (showValueIndicator) {
         _state.valueIndicatorController.forward();
-        if (_state.interactionTimer.isActive) {
-          _state.interactionTimer.cancel();
-        }
+        _state.interactionTimer?.cancel();
         _state.interactionTimer = new Timer(_minimumInteractionTime * timeDilation, () {
-          if (!_active && _state.valueIndicatorController.status == AnimationStatus.completed) {
+          _state.interactionTimer = null;
+          if (!_active &&
+              _state.mounted && // Don't want to start a disposed animation.
```

Comment:
since you cancel the timer on dispose, in theory you shouldn't be able to get here with mounted==false; did you add this in defense or did you run into this case?

Reply:
Yeah, it was defensive, but you're right.  Removed.